e2e-tests/IIoTPlatform-E2E-Tests/Standalone/A_PublishSingleNodeStandaloneTestTheory.cs

```diff
@@ -16,92 +16,89 @@ namespace IIoTPlatform_E2E_Tests.Standalone {
             Assert.True(result);
         }
 
         [Fact, PriorityOrder(3)]
-        public async Task Test_StartPublishingSingleNode_Expect_Success() {
+        public void Test_StartPublishingSingleNode_Expect_Success() {
             var cts = new CancellationTokenSource(TestConstants.MaxTestTimeoutMilliseconds);
 
-            var simulatedPublishedNodesConfiguration = await TestHelper.GetSimulatedPublishedNodesConfigurationAsync(_context, cts.Token);
+            var simulatedPublishedNodesConfiguration = TestHelper.GetSimulatedPublishedNodesConfigurationAsync(_context, cts.Token).GetAwaiter().GetResult();
 
             var model = simulatedPublishedNodesConfiguration[simulatedPublishedNodesConfiguration.Keys.First()];
             model.OpcNodes = model.OpcNodes.Take(1).ToArray();
 
-            await TestHelper.SwitchToStandaloneModeAndPublishNodesAsync(new[] { model }, _context, cts.Token);
+            TestHelper.SwitchToStandaloneModeAndPublishNodesAsync(new[] { model }, _context, cts.Token).GetAwaiter().GetResult();
+
+            Task.Delay(60 * 1000, cts.Token).GetAwaiter().GetResult(); //wait some time till the updated pn.json is reflected
         }
 
         [Fact, PriorityOrder(4)]
-        public async Task Test_WaitForModuleDeployed() {
+        public void Test_WaitForModuleDeployed() {
             var cts = new CancellationTokenSource(TestConstants.MaxTestTimeoutMilliseconds);
 
             // We will wait for module to be deployed.
-            var exception = await Record.ExceptionAsync(async () => await _context.RegistryHelper.WaitForIIoTModulesConnectedAsync(_context.DeviceConfig.DeviceId, cts.Token, new string[] { "publisher_standalone" }));
+            var exception = Record.Exception(() => _context.RegistryHelper.WaitForIIoTModulesConnectedAsync(_context.DeviceConfig.DeviceId, cts.Token, new string[] { "publisher_standalone" }).GetAwaiter().GetResult());
             Assert.Null(exception);
         }
 
         [Fact, PriorityOrder(5)]
-        public async Task Test_VerifyDataAvailableAtIoTHub_Expect_NumberOfValueChanges_GreaterThan_Zero() {
+        public void Test_VerifyDataAvailableAtIoTHub_Expect_NumberOfValueChanges_GreaterThan_Zero() {
             var cts = new CancellationTokenSource(TestConstants.MaxTestTimeoutMilliseconds);
 
             //use test event processor to verify data send to IoT Hub (expected* set to zero as data gap analysis is not part of this test case)
-            await TestHelper.StartMonitoringIncomingMessagesAsync(_context, 0, 0, 0, cts.Token);
+            TestHelper.StartMonitoringIncomingMessagesAsync(_context, 0, 0, 0, cts.Token).GetAwaiter().GetResult();
 
             // wait some time to generate events to process
-            await Task.Delay(90 * 1000, cts.Token);
-            var json = await TestHelper.StopMonitoringIncomingMessagesAsync(_context, cts.Token);
+            Task.Delay(90 * 1000, cts.Token).GetAwaiter().GetResult();
```

Comment:
>90 * 1000 [](start = 23, length = 9)

magic number

Reply:
no, this is the default timeout, changed to use constant TestConstants.DefaultTimeoutInMilliseconds